app/src/main/java/com/example/triptracker/view/home/HomeScreen.kt

```diff
@@ -390,7 +390,7 @@ fun DisplayItineraries(
               },
               displayImage = true,
               test = test,
-          )
+              navigation = navigation)
```

Comment:
We will have a small merge conflict to handle because in #249 I also modified the arguments of DisplayItineraries